app/pages/admin/user-settings.jsx

```diff
@@ -64,6 +85,17 @@ class UserSettings extends Component {
         <br />
         <UserResources type="projects" user={this.state.editUser} />
         <UserResources type="organizations" user={this.state.editUser} />
+        <h4>Classification history</h4>
+        <p>Total classifications: {this.state.totalClassifications}</p>
+        <ul>
+        {this.state.ribbonData.map(project => (
+          <li>
```

Comment:
Not blocking, but should this have a `key`?

Reply:
Yeah, I think it should.